### ISSUE TITLE
[BUG] - Can't submit PIREP to TFDi Error

### ISSUE THREAD

**Issue by walkerairtrans** [invernyx/smartcars-3-bugs#168]
### Describe the bug

Getting a red toast when submitting a PIREP that it cannot submit to your server (assuming a premium service or something.)

Flight still posts to VA server though.

### How do you reproduce this bug?

1. Submit a flight
2. Watch for the red toast notification

### Expected behavior

Disable that notification if it's not in deployment maybe?

### Operating system

Windows 11

### Community airline

Walker Air Transport

### smartCARS Version

Latest

### Plugins installed

chat, map, flight center, flight tracker, logbook

**Comment by bgiorgio0506** [comment]
@walkerairtrans  can you send me the log in %localappdata%\TFDi Design\smartCARS 3\logs

**Comment by walkerairtrans** [comment]
> %localappdata%\TFDi Design\smartCARS 3\logs

Here you go -- I am not signed into my TFDi account.

```
error: Could not register your flight on your TFDi Design account, failed with error code: 404 and message: Flight not found {"code":404,"name":"HttpClientError","response":{"message":"Flight not found"},"sourceUri":"https://smartcars.tfdidesign.com/v3/api/flight/complete","timestamp":"2023-05-04T00:44:15.858Z"}
error: Failed to call local API {"endpoint":"api/com.tfdidesign.flight-tracking/complete","error":{"code":"ERR_BAD_RESPONSE","config":{"adapter":["xhr","http"],"data":"{}","env":{},"headers":{"Accept":"application/json, text/plain, */*","Content-Type":"application/json"},"maxBodyLength":-1,"maxContentLength":-1,"method":"get","params":{},"timeout":0,"transformRequest":[null],"transformResponse":[null],"transitional":{"clarifyTimeoutError":false,"forcedJSONParsing":true,"silentJSONParsing":true},"url":"http://localhost:7172/api/com.tfdidesign.flight-tracking/complete","xsrfCookieName":"XSRF-TOKEN","xsrfHeaderName":"X-XSRF-TOKEN"},"message":"Request failed with status code 500","name":"AxiosError","stack":"AxiosError: Request failed with status code 500\n    at http://localhost:7172/plugins/com.tfdidesign.flight-tracking/static/js/main.47647d2e.js:2:243683\n    at XMLHttpRequest.d (http://localhost:7172/plugins/com.tfdidesign.flight-tracking/static/js/main.47647d2e.js:2:243831)","status":500},"timestamp":"2023-05-04T00:44:15.860Z"}
```

**Comment by bgiorgio0506** [comment]
I cannot reproduce your bug locally can you provide exact steps.

**Comment by walkerairtrans** [comment]
> I cannot reproduce your bug locally can you provide exact steps.

I am at the F1 race in Miami so it won’t be until I’m back. But I am not signed into my TFDi account and neither is one of my beta testers and he gets the same error popping up. 

I’ll send more details when I’m back home.

**Comment by turbofandude** [comment]
Fixed in 0.8.3
